config/software/datadog-agent-integrations.rb

```diff
@@ -0,0 +1,67 @@
+require './lib/ostools.rb'
+
+name 'datadog-agent-integrations'
+
```

Comment:
This is a part of building the datadog-agent. In the definition of the independent checks we do declare it: https://github.com/DataDog/dd-agent-omnibus/blob/master/resources/datadog-integrations/project.rb.erb#L47